cmd/ingester/main.go

```diff
@@ -77,7 +77,7 @@ func main() {
 	}
 	defer chunkStore.Stop()
 
-	ingester, err := ingester.New(ingesterConfig, chunkStore)
+	ingester, err := ingester.New(ingesterConfig, schemaConfig.CreationGracePeriod, chunkStore)
```

Comment:
Rather than having to pass a lone variable into `ingester.New`, would it make sense to just add CreationGracePeriod to ingesterConfig and keep the old signature?

Reply:
I'm struggling to see how to do that and keep flag parsing sane.
We can't have the same flag registered twice.
Ingester takes an `ingester.Config` only; TableManager takes a `chunk.SchemaConfig` only.

Reply:
Perhaps just `ingesterConfig.CreationGracePeriod = schemaConfig.CreationGracePeriod` on the line above? I don't feel too strongly about needing to remove the extra argument though if that does not feel clean to you.

Reply:
The extra arg means I don't forget to do that (e.g. in `lite` which I did forget first try)

Reply:
Good point. :shipit: